R/flow_denovomodule.R

```diff
@@ -308,7 +337,7 @@ denovo <- function(input,
           ))
           return(NULL)
         } 
-        mygenome <- BSgenome.Hsapiens.UCSC.hg38
+        mygenome <- getBSgenome("BSgenome.Hsapiens.UCSC.hg38")
```

Comment:
nao deveria ter um else if, se for outro genoma aqui ?
ai carrega direto com getBSgenome